src/components/questions/question/Question.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import QuestionDetails from './QuestionDetails'
+import QuestionTags from './QuestionTags'
+
+import withCSS from '../../../lib/withCSS'
+
+const Question = ({ head, id, lastUsed, tags, title, type, version }) =>
+  (<div className="container">
+    {head}
+
+    <h2 className="title">
+      #{id.substring(0, 7)} - {title} {version > 1 && `(v${version})`}
+    </h2>
+    <div className="tags">
+      <QuestionTags tags={tags} type={type} />
+    </div>
+    <div className="details">
+      <QuestionDetails lastUsed={lastUsed} />
+    </div>
+
+    <style jsx>{`
```

Comment:
We need to be consistent with the handling in the jsx tag. A space between CSS classes or not? :)